python/lsst/ip/isr/isrTask.py

```diff
@@ -1783,7 +1783,8 @@ def run(self, ccdExposure, *, camera=None, bias=None, linearizer=None,
         # calculate additional statistics.
         outputStatistics = None
         if self.config.doCalculateStatistics:
-            outputStatistics = self.isrStats.run(ccdExposure, overscanResults=overscans,
+            outputStatistics = self.isrStats.run(ccdExposure, serialOverscanResults=overscans,
+                                                 parallelOverscanResults=[None for _ in overscans],
```

Comment:
The parallelOverscanResults could/should just be None here (but as asked above this seems to be unused anyway).